src/controllers/AuthController.js

```diff
@@ -0,0 +1,67 @@
+const logErrorHelper = require('../logger/loggerHelper');
+export default class AuthController {
+    constructor(tokenList, usersList, authService) {
+        this.tokenList = tokenList;
+        this.usersList = usersList;
+        this.authService = authService;
+    }
+
+    async login(req, res, next) {
+        try {
+            const user = await this.usersList.findOne({
+                where: {
+                    login: req.body.login
+                },
+                raw: true
+            });
+
+            if (!user) {
+                return res.status(404).json(`${req.body.login} does not exist. Please register first!`);
+            }
+            const match = await this.authService.compare(req.body.password, user.password);
+            if (match) {
+                return res.send(this.updateTokens(user.id));
+            } else {
```

Comment:
@lenon777, the `else` branch is not required here, because you have `return` in `then` branch